src/main/java/uk/gov/justice/hmpps/prison/repository/StaffRepository.java

```diff
@@ -24,9 +25,6 @@
 @Repository
 @Slf4j
 public class StaffRepository extends RepositoryBase {
-    private static final String NAME_FILTER_QUERY_TEMPLATE = " AND (UPPER(FIRST_NAME) LIKE '%s%%' OR UPPER(LAST_NAME) LIKE '%s%%')";
```

Comment:
so what did you do about these?

Reply:
any contstants that contained sql were moved to the relevant enum class. this one is now `StaffRepositorySql.NAME_FILTER_QUERY_TEMPLATE`